Gemfile

```diff
@@ -11,6 +11,7 @@ source 'https://rubygems.org'
 # gem 'cocoapods', git: "https://github.com/CocoaPods/CocoaPods.git", ref: "9cebcde577f56aa26f27d8aa501b51fdd4d6abdb"
 # gem 'cocoapods-core', git: "https://github.com/CocoaPods/Core.git", ref: "f7cf05720eab935d7d50e35224d263952176fb53"
 # gem 'xcodeproj', git: "https://github.com/CocoaPods/Xcodeproj.git", ref: "eeccae7275645753cbaf45d96fc4b23e4b8b3b9f"
+# This line will trip CI.
```

Comment:
Remove before merging

Reply:
Done in 54bf31a25e8a2c86f107755a8812efbb3aa95ca